examples/tutorial.dx

```diff
@@ -88,7 +88,7 @@ import plot
 ' This style of using `for` to construct type-inferred tables is central to what
   makes Dex powerful. Tables do not only need to have `Fin` types.
 
-' Let's consider another example. This one produces a list of
+' Let's consider another example. This one produces a list of 1s
```

Comment:
Minor personal preference: how about spelling out "ones"?
I tend to spell out "zeros" and "ones" in writing until the numbers get big (like 24 below).

```suggestion
' Let's consider another example. This one produces a list of ones
```